projects/packages/my-jetpack/_inc/components/product-cards-section/index.tsx

```diff
@@ -23,7 +24,7 @@ type DisplayItemsProps = {
 
 type DisplayItemType = Record<
 	// We don't have a card for Security or Extras, and scan is displayed as protect.
-	Exclude< JetpackModule, 'extras' | 'scan' | 'security' >,
+	Exclude< JetpackModule, 'extras' | 'scan' | 'security' | 'ai' >,
```

Comment:
Let's update the comment above as well. Currently it's a bit confusing because in a way "ai" card is displayed (I'm aware it's  about reducing redundancy of "jetpack-ai" and "ai" - but I might forget it in ~3 months 😅)